kolibri/plugins/learn/assets/src/views/learn-page/index.vue

```diff
@@ -36,27 +41,76 @@
 <script>
 
   import { PageNames } from '../../constants';
-  import { getCurrentChannelObject } from 'kolibri.coreVue.vuex.getters';
   import pageHeader from '../page-header';
   import contentCardCarousel from '../content-card-carousel';
+
+  const carouselLimit = 15;
+
   export default {
     $trNameSpace: 'learnPageIndex',
     $trs: {
-      pageHeader: 'Recommended',
       popularSectionHeader: 'Most popular',
       suggestedNextStepsSectionHeader: 'Next steps',
       resumeSectionHeader: 'Resume',
-      popularSectionSubHeader: '{numOfItems, number} popular items',
-      suggestedNextStepsSectionSubHeader: '{numOfItems, number} suggested items',
-      resumeSectionSubHeader: '{numOfItems, number} items to be resumed',
-      allContentSectionHeader: 'All items',
+      overviewSectionHeader: 'Overview',
+      sectionSubHeader: '{numOfItems, plural, one {# item} other {# items}}',
+      subsetSectionSubHeader: '{subset, number} of {total, plural, one {# item} other {# items}}',
```

Comment:
New subheader, to be used when we need a `view more` link in the header. @khangmach